test/functional/test_site/sub_site/testReuse.md

```diff
@@ -19,5 +19,8 @@ PIC tags:
 <pic src="{{baseUrl}}/images/I'm not allowed to use my favorite tool.png"></pic>
 <pic src="https://dummyimage.com/600x400/000/fff"></pic>
 
-Anchor link:
-<a href="https://dummyimage.com/600x400/000/fff">External Image</a>
\ No newline at end of file
+Anchor:
+<a href="https://dummyimage.com/600x400/000/fff">External Image</a>
+<a href="{{baseUrl}}/images/I'm not allowed to use my favorite tool.png">Link to picture</a>
+<a id="namedAnchor">Named Anchor</a>
+<a>Useless anchor</a>
```

Comment:
Could we have a less emotional name for this? 😅 
`Anchor with no attributes` perhaps?